extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,141 @@
+# KHR_draco_mesh_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+* Kai Ninomiya, Google, <mailto:kainino@google.com>
+* Patrick Cozzi, Cesium, [@pjcozzi](https://twitter.com/pjcozzi)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use [Draco geometry compression](https://github.com/google/draco) libraries in glTF format. This allows glTF to support streaming compressed geometry data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+Draco geometry compression library could be used for `primitive` by adding an `extension` property to a primitive, and defining its `KHR_draco_mesh_compression` property.
+
+The following picture shows the structure of the schema update. 
+
+**Figure 1**: Structure of geometry compression extension.
+![](figures/structure.png)
+
+In general, we will use the extension to point to the buffer that contains the compressed data. The major change is that the `accessor` in an extended `primitive` no
+longer points to a `bufferView`. Instead, the `attributes` of a primitive will use the decompressed data. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present, it will be used with `sparse` field to act as a sparse accessor. In this extension, we will ignore the `bufferView` property.
+
+Usage of the extension must be listed in the `extensionUsed` and `extensionsRequired`. 
```

Comment:
Merging threads from above, how about this?

```
// primitive, with all attributes referencing uncompressed accessors
"primitive" {
  "attributes" : {
    "POSITION" : 11,
    "NORMAL" : 12,
    "TEXCOORD_0" : 13,
    "TEXCOORD_1" : 14
  },
  ...
}
```

```
// extension
"KHR_draco_mesh_compression" : {
  "bufferView" : 5,
  "attributes": ["POSITION", "NORMAL", "TEXCOORD_0"]
}
```

In this case, the Draco extension is not required. A loader observing that three attributes are available in compressed form may choose not to download the uncompressed versions. A loader that doesn't support the extension may parse the `attributes` as usual. It also allows additional, uncompressed attributes to be used alongside what Draco provides, as `TEXCOORD_1` above.

This is really just two small tweaks to what @fanzhanggoogle has already written:

1. If attributes are _not_ available in uncompressed form, they should not be included in `primitive.attributes`. Draco must be considered a required extension, if any attributes are exclusively available in compressed form.
2. Rename `attributesOrder` to `attributes`, as the order isn't needed.

I think this would allow efficient fallbacks as you mention @bghgary, if we strongly encourage exporters to put the Draco data and the non-Draco fallback data into separate binary sources.